lib/rom/support/inflector.rb

```diff
@@ -3,45 +3,65 @@ module ROM
   #
   # @private
   module Inflector
-    begin
-      @inflector =
-        begin
-          require 'active_support/inflector'
-          ::ActiveSupport::Inflector
-        rescue LoadError
-          require 'inflecto'
-          ::Inflecto
-        end
+    BACKENDS = {
+      activesupport: ['active_support/inflector', '::ActiveSupport::Inflector'],
+      inflecto: ['inflecto', '::Inflecto']
+    }.freeze
+
+    def self.realize_backend(path, inflector_class)
+      require path
+      Object.const_get(inflector_class)
```

Comment:
That was there out of force of habit I guess!  I'll drop the leading `::`.  Seeing as we're going from `Object` that should be fine anyway.